pages/hugo.yml

```diff
@@ -30,19 +30,28 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      HUGO_VERSION: 0.102.3
+      HUGO_VERSION: 0.108.0
+      EMBEDDED_DART_SASS_VERSION: 1.56.2
     steps:
       - name: Install Hugo CLI
         run: |
-          wget -O ${{ runner.temp }}/hugo.deb https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_Linux-64bit.deb \
+          wget -O ${{ runner.temp }}/hugo.deb https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_linux-amd64.deb \
           && sudo dpkg -i ${{ runner.temp }}/hugo.deb
+      - name: Install Dart Sass Embedded
```

Comment:
👋 Hello and thank you for your contribution.

Could we make this step optional if `EMBEDDED_DART_SASS_VERSION` is empty (and add a comment about that line 34)?

I am not sure how Dart Sass Embedded is popular nowadays with Hugo but making the starter workflow as simple to customize as possible for people that may not be too familiar with Actions is something important.

Reply:
> I am not sure how Dart Sass Embedded is popular nowadays with Hugo

With LibSass now deprecated, usage will continue to increase. Over time, independent theme creators will update their themes to use the Dart Sass transpiler. So a new user, making use of a prebuilt theme, will not have a good experience.

I will make the step optional if EMBEDDED_DART_SASS_VERSION is empty, but it should not empty by default. This should just... work.